src/java/org/apache/cassandra/db/virtual/SettingsTable.java

```diff
@@ -49,6 +49,13 @@
               .filter(f -> !Modifier.isStatic(f.getModifiers()))
               .collect(Collectors.toMap(Field::getName, Functions.identity()));
 
+    @VisibleForTesting
+    static final Map<String, Field> ANNOTATED_FIELDS =
+    Arrays.stream(Config.class.getFields())
```

Comment:
nit: wrapping is a bit off here. can you wrap with the operator, and indent a bit more , pls.

Reply:
It was following the same format as FIELDS and overrides but I agree what you said looks better. Should I reformat the other 2? Asking as I know you don't like when we do unrelated changes but I think it will be nice for consistency

Reply:
For the record, I changed this one locally to:
```
@VisibleForTesting
    static final Map<String, Field> ANNOTATED_FIELDS = Arrays.stream(Config.class.getFields())
                                                             .filter(f -> !Modifier.isStatic(f.getModifiers()))
                                                             .filter(f -> f.isAnnotationPresent(Replaces.class))
                                                             .collect(Collectors.toMap(Field::getName, Functions.identity()));
```

Reply:
Right. If the style has precedence in this file then please leave it.